src/types.ts

```diff
@@ -234,6 +234,19 @@ export interface Segment {
 
 export type SegmentType = "sprint" | "climb" | "segment";
 
+export interface Bike {
+  make: string;
+  name: string;
+  year?: number;
+  level: number; // negative for special cases like Tron bike
+  frameWeight: number;
```

Comment:
Let's add a jsdoc comment about the unit of the weights. It's obvious from the numbers but with the types it could be kg or gramms.